libs/cardano-ledger-api/src/Cardano/Ledger/Api/Tx/Out.hs

```diff
@@ -43,45 +48,86 @@ module Cardano.Ledger.Api.Tx.Out (
   -- * Babbage Era
   BabbageTxOut,
   BabbageEraTxOut (..),
-  setMinCoinTxOut,
-  setMinCoinSizedTxOut,
 )
 where
 
 import Cardano.Ledger.Alonzo.TxBody (AlonzoEraTxOut (..), AlonzoTxOut)
 import Cardano.Ledger.Api.Scripts.Data
 import Cardano.Ledger.Babbage.TxBody (BabbageEraTxOut (..), BabbageTxOut)
 import Cardano.Ledger.Binary
+import Cardano.Ledger.Coin
 import Cardano.Ledger.Core (EraTxOut (..), PParams, coinTxOutL, eraProtVerLow)
 import Cardano.Ledger.Shelley.TxBody (ShelleyTxOut)
 import Lens.Micro
 
--- | Same as `setMinCoinSizedTxOut`, except it doesn't require the size of the
--- TxOut and will recompute it if needed. Initial amount is not important.
-setMinCoinTxOut :: EraTxOut era => PParams era -> TxOut era -> TxOut era
-setMinCoinTxOut pp = go
+setMinCoinTxOutInternal ::
+  EraTxOut era =>
+  (Coin -> Coin -> Bool) ->
+  PParams era ->
+  TxOut era ->
+  TxOut era
+setMinCoinTxOutInternal f pp = go
   where
-    go txOut =
+    go !txOut =
       let curMinCoin = getMinCoinTxOut pp txOut
           curCoin = txOut ^. coinTxOutL
-       in if curCoin == curMinCoin
+       in if curCoin `f` curMinCoin
```

Comment:
sure you can't be tempted by some fun and obscure unicode symbol? :laughing:

Reply:
I don't think it is possible to have infix operators as function argument names .

Reply:
of well, `f` is a great name too :grin: